test/cookbook_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ingredient'
+require './lib/recipe'
+require './lib/cookbook'
+require './lib/pantry'
+require 'pry'
+
+class RecipeTest < MiniTest::Test
+
+  def setup
+    @pantry = Pantry.new
+    @cookbook = CookBook.new
+    @ingredient1 = Ingredient.new({name: "Cheese", unit: "C", calories: 100})
+    @ingredient2 = Ingredient.new({name: "Macaroni", unit: "oz", calories: 30})
+    @ingredient3 = Ingredient.new({name: "Ground Beef", unit: "oz", calories: 100})
+    @ingredient4 = Ingredient.new({name: "Bun", unit: "g", calories: 75})
+    @recipe1 = Recipe.new("Mac and Cheese")
+    @recipe2 = Recipe.new("Cheese Burger")
+  end
+
+  def test_cook_book_recipes
+    @cookbook.add_recipe(@recipe1)
+    @cookbook.add_recipe(@recipe2)
+
+    assert_equal [@recipe1, @recipe2], @cookbook.recipes
+  end
+
+  def test_the_calories_in_recipe
```

Comment:
Make sure each method gets it's own single test. There are multiple methods being tested in this one single test. We want to test each one individually.